enh(rn): complete the inherited downtimes bugfix (#1395)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-21.10/releases/centreon-core.md

- Fixed an issue causing BAM Business Activities (best status) to remain in an OK state when the OK KPIs were removed
- Refactored the BAM Business activities downtimes inheritance mechanism so that they are properly inherited and not duplicated anymore.

> > You may have remaining downtimes because they were duplicated from a KPI downtime that ended before the update. In t

Reviewer: ```suggestion
> You may have remaining downtimes because they were duplicated from a KPI downtime that ended before the update. In that case, you will have to apply this procedure.
```

> hat case, you will have to apply this procedure.

Reviewer: ```suggestion
```

```bash
systemctl stop centengine
sed -i -zE 's/(servicecomment|servicedowntime) \{\nhost_name=_Module_BAM_1\n[^}]*\}\n//g' /var/log/centreon-engine/retention.dat
systemctl start centengine
```

> All the the downtimes applied on Business Activities have now been removed.

Reviewer: ```suggestion
All the downtimes applied on Business Activities have now been removed.
```

> You must then restart `centengine` service on all pollers to restore the legitimate inherited downtimes.

Reviewer: ```suggestion
You must then restart the `centengine` service on all pollers to restore the legitimate inherited downtimes.
```
